debian/jitsi-meet-prosody.postinst

```diff
@@ -184,6 +175,11 @@ case "$1" in
             PROSODY_CONFIG_PRESENT="false"
         fi
 
+        if ! grep -q -- 'component_admins_as_room_owners = ' $PROSODY_HOST_CONFIG ;then
```

Comment:
Why is this necessary? Maybe leave a comment.

Reply:
Ok, I will add a comment.